examples/ui-demo/src/components/configuration/Styling.tsx

```diff
@@ -9,9 +9,17 @@ import { ThemeSwitch } from "../shared/ThemeSwitch";
 import { ColorPicker } from "./ColorPicker";
 import { PhotoUploads } from "./PhotoUpload";
 import { PaletteIcon } from "../icons/palette";
+import { useState } from "react";
 
 export function Styling({ className }: { className?: string }) {
   const { config, setConfig } = useConfig();
+  const [supportURL, setSupportURL] = useState("");
+  const handleChangeSupportURL = () => {
```

Comment:
useCallback? or useMemo?

Reply:
IMO this is so minor the garbage collection could be more of a perforamance hit than memoizing this

Reply:
Yeah fair also TIL. Let's ship this without useCallback/memo